contracts/periphery/astETH.sol

```diff
@@ -0,0 +1,34 @@
+import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+contract astETH is ERC20 {
+    using SafeERC20 for IERC20;
+
+    address public immutable stETH;
+    address public immutable feeRecipient;
```

Comment:
Do we want `feeRecipient` to be immutable? Other fee recipients (ie in Vault) are changeable. I could see an argument for allowing the same here, or simply calling to `vault.feeRecipient()` to piggyback on whatever is set there.